src/components/v2v3/V2V3Project/ProjectSafeDashboard/ExecutedSafeTransactionsListing.tsx

```diff
@@ -24,7 +24,7 @@ export function ExecutedSafeTransactionsListing({
         (transaction: SafeTransactionType, idx: number) => (
           <SafeTransaction
             key={`safe-${transaction.nonce}-${idx}`}
-            transaction={transaction}
+            transaction={{ ...transaction, isHistory: true }}
```

Comment:
yeah so maybe we can just make `isHistory` some kind of prop.

Reply:
re: https://github.com/jbx-protocol/juice-interface/pull/2126#discussion_r987350269